extensions/data-workspace/src/test/projectProviderRegistry.test.ts

```diff
@@ -54,14 +55,24 @@ suite('ProjectProviderRegistry Tests', function (): void {
 				description: ''
 			}
 		],
-		[{
-			id: 'ta1',
-			run: async (): Promise<any> => { return Promise.resolve(); }
-		},
-		{
-			id: 'ta2',
-			run: async (): Promise<any> => { return Promise.resolve(); }
-		}]);
+			[{
+				id: 'ta1',
+				run: async (): Promise<any> => { return Promise.resolve(); }
+			},
+			{
+				id: 'ta2',
+				run: async (): Promise<any> => { return Promise.resolve(); }
+			}],
+			[{
+				name: 'ti1',
```

Comment:
does ti stand for table information? It would be helpful if the names were a little more descriptive, like tableInfo1, so it's easy to tell what they are for at a glance. Same for ta1 and ta2 above